apps/mobile/src/utils/types.ts

```diff
@@ -1,5 +1,7 @@
 import { ReactNode } from 'react';
 
+export type InputCurrencyMode = 'crypto' | 'fiat';
```

Comment:
Maybe this should be in `models` ? I'm not sure though . It could be more `CurrencyFormat` or something like that

Reply:
This is an app-specific type that will be bound to user's implicit preference. I don't think it's in any way related to models.